war/src/main/resources/templates/scores/details.html

```diff
@@ -108,9 +108,19 @@ <h3>Probes results</h3>
             </tr>
             </thead>
             <tbody>
-            <tr data-th-each="item: ${plugin.details}" data-th-with="probeResult = ${item.value}">
-                <td><code data-th-text="${probeResult.id}" class="label"></code></td>
-                <td data-th-text="${probeResult.message}"></td>
+             <tr data-th-each="item: ${plugin.details}" data-th-with="probeResult = ${item.value}">
+              <td><code data-th-text="${probeResult.id}" class="label"></code></td>
+              <td>
+              <td>
+                <span data-th-if="${probeResult.id == 'security'}">
+                  <span data-th-each="part : ${#strings.arraySplit(probeResult.message, ',')}" data-th-inline="text">
+                    <span data-th-with="parts=${#strings.arraySplit(part, ' ')}">
+                        <a data-th-text="${parts[0]}" data-th-href="${parts[1]}"></a>
+                    </span>
+                   </span>
+                </span>
```

Comment:
Doesn't work when there is no security issue.

Reply:
Added one more condition @alecharp 
<img width="725" alt="Screenshot 2024-04-17 at 10 42 49 PM" src="https://github.com/jenkins-infra/plugin-health-scoring/assets/60972989/1a992e3a-aa5b-4394-b3dc-e3fa56606863">

Reply:
<img width="1525" alt="image" src="https://github.com/jenkins-infra/plugin-health-scoring/assets/60972989/9954d230-0f4e-4ca6-bbbb-ab769e1cb314">
Security link